### PR TITLE
awscli

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+RUN dnf install -y \
+    unzip-6.0-56.el9.x86_64 \
+    wget-1.21.1-7.el9.x86_64 \
+    dnf-plugins-core-4.1.0-3.el9.noarch \
+    make-1:4.3-7.el9.x86_64 && \
     dnf config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo && \
     dnf install -y terraform-1.3.7-1.x86_64 && \
-    curl https://github.com/gruntwork-io/terragrunt/releases/download/v0.43.0/terragrunt_linux_amd64 -o /usr/local/bin/terragrunt && \
+    dnf clean all -y && \
+    wget https://github.com/gruntwork-io/terragrunt/releases/download/v0.43.0/terragrunt_linux_amd64 > /usr/local/bin/terragrunt && \
```
Comment (RajeshKumarSrivastava) [inline Dockerfile:11]:
```suggestion
    wget https://github.com/gruntwork-io/terragrunt/releases/download/v0.43.0/terragrunt_linux_amd64  -O /usr/local/bin/terragrunt && \
```

Comment (lmilbaum) [reply Dockerfile:11]: I wonder. Should we keep the `curl` or `wget`? WDYT?

Comment (RajeshKumarSrivastava) [reply Dockerfile:11]: IMO `wget` is more than enough for our requirement here

Comment (lmilbaum) [reply Dockerfile:11]: OK. Rebasing and fixing the PR.

**Review by RajeshKumarSrivastava — APPROVED** [review]
LGTM

**Pull request merged**